README.md

```diff
@@ -44,12 +44,28 @@ $ cd cozy-drive
 $ yarn install
 ```
 
-:pushpin: If you use a node environment wrapper like [nvm] or [ndenv], don't forget to set your local node version `8` before doing a `yarn install`.
+:pushpin: If you use a node environment wrapper like [nvm] or [ndenv], don't forget to set your local node version `12` before doing a `yarn install`.
 
-:warning: During its early ages, _cozy-drive_ uses beta versions of [cozy-ui] and [cozy-client-js], take a look at the ["living on the edge" note](#living-on-the-edge) below to know hot to install and configure the latest available versions.
+:warning: During its early ages, _cozy-drive_ uses beta versions of [cozy-ui] and [cozy-client-js], take a look at the ["living on the edge" note](#living-on-the-edge) below to know how to install and configure the latest available versions.
 
 Cozy's apps use a standard set of _npm scripts_ to run common tasks, like watch, lint, test, build…
 
+Run in dev mode:
+```sh
+$ yarn watch:drive:browser
+```
+
+Run in build mode:
+```sh
+$ yarn start:drive:browser
+```
+
+Build the app and run in cozy-stack:
+```sh
+$ yarn build:drive
+$ cd ./<path_de_ton_dossier_de_dev>/cozy-stack  && cozy-stack serve --appdir drive:/<path_de_ton_dossier_de_dev>/cozy-drive/build/drive
```

Comment:
If installed correctly, you shouldn't have to `cd` on cozy-stack dir to execute it.
```suggestion
$ cozy-stack serve --appdir drive:/<project_absolute_path>/cozy-drive/build/drive
```